plsql/PlSqlParser.g4

```diff
@@ -561,9 +561,68 @@ create_table
 // Column_properties clause goes here
 // Partition clause goes here
 // Many more varations to capture
+       (
+         table_range_partition_by_clause
+       )?
+
+       ( (ENABLE | DISABLE)? ROW MOVEMENT )?
+
+       ( FLASHBACK ARCHIVE flashback_archive=REGULAR_ID
+       | NO FLASHBACK ARCHIVE
+       )?
+
+      ( AS subquery )?
+
       SEMICOLON
```

Comment:
I suggest using string literals for primitive tokens: SEMICOLON -> ';' COMMA -> ',' LEFT_PAREN -> '(' RIGHT_PAREN -> ')'.

Reply:
I assume the comment is directed at the SEMICOLON here.  (AS subquery)?  is  an optional clause.  Asking that because the (AS subquery)? is in green.

Reply:
Yep, SEMICOLON :)